terraform/manager.tf

```diff
@@ -235,7 +235,11 @@ write_files:
       fi
     path: /root/manager.sh
     permissions: 0700
+packages:
+  - docker-ce-cli
+  - docker-compose
 runcmd:
+  - "dpkg -r ansible || true"
```

Comment:
Dito.

Reply:
Sure?
The OTC Ubuntu images come with ansible-2.9 preinstalled and that's a problem for the 2.10+ installation.
So I believe, this snippet will be needed in the future.

Reply:
Ok. Makes sense then. Please add as an independent PR.

Reply:
This is currently in a separate patch file in PR #594 (ansible-otc.patch).
It's harmless on images where ansible is not preinstalled, but it's still an OTC specific workaround, so I wonder whether it's acceptable to have included always.

Reply:
It does not interfere if Ansible is not installed and removes it if it is. Fits.